### PR TITLE
Add anchors to pages

### PR DESCRIPTION
**Pull request by pikinier20**
Added anchors for:
- headers - headers title lowercased and without whitespaces e.g. "See also" -> "#seealso"
- symbols - symbols DRI

### CODE REVIEW

File: plugins/base/src/main/kotlin/translators/documentables/PageContentBuilder.kt [hunk 1]
```diff
@@ -73,14 +73,26 @@ open class PageContentBuilder(
 
         fun header(
             level: Int,
+            text: String,
             kind: Kind = ContentKind.Main,
             styles: Set<Style> = mainStyles,
             extra: PropertyContainer<ContentNode> = mainExtra,
             block: DocumentableContentBuilder.() -> Unit
```
Comment (sellophane) [inline plugins/base/src/main/kotlin/translators/documentables/PageContentBuilder.kt:80]: Add a default value here for block, we will omit multiple `{ }` this way

**Comment by Kordyjan** [comment]
Please rebase it and regenerate expect tests.

**Comment by pikinier20** [comment]
Rebased

**Comment by pikinier20** [comment]
Rebased.

**Comment by pikinier20** [comment]
Rebased.

**Pull request merged**